### DISCUSSION TITLE
Live / Reaper / Ninjam

### DISCUSSION THREAD

**Discussion by microcreators** [ExistentialAudio/BlackHole#144] [General]
Just got a working "online jamming" setup via Ableton Live + Reaper + Ninjam. All on damn Catalina!  👓  Thanks! 
If anyone's interested, I can post a simple how-to.

ps. now I just need my 32-bit vst back, but I'm guessing this not gonna happen :)

**Comment by NicoInfinito** [comment]
Hey bro, please how do you do
thanks

**Comment by microcreators** [comment]
i'll post something over the weekend

**Comment by NicoInfinito** [comment]
Thanks my friend!!!

**Comment by microcreators** [comment]
Here you go... https://github.com/microcreators/raln
Done on a train in 60 mins, so just text + 2 example project files that should make things easier. Hope it helps.
